blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -92,4 +92,5 @@ export const State = Record({
     steps: null,
     currentBranches: null,
     testResults: null,
+    capabilities: null,
```

Comment:
No, not here. The component keeps track of the capabilities for each object type. The component maintains it's own render state based on whether or not it's been fetched.